packages/passport/sdk/src/guardian/guardian.test.ts

```diff
@@ -38,15 +39,29 @@ describe('Guardian', () => {
         logoutRedirectUri: 'http://localhost:3000/logout',
         redirectUri: 'http://localhost:3000/redirect',
       }),
+      authManager: { getUser: jest.fn().mockResolvedValue(mockUserImx) } as unknown as AuthManager,
     });
   });
 
   describe('evaluateImxTransaction', () => {
+    afterEach(jest.resetAllMocks);
+    beforeEach(() => {
+      guardianClient = new GuardianClient({
+        confirmationScreen: mockConfirmationScreen,
+        config: new PassportConfiguration({
+          baseConfig: {} as ImmutableConfiguration,
+          clientId: 'client123',
+          logoutRedirectUri: 'http://localhost:3000/logout',
+          redirectUri: 'http://localhost:3000/redirect',
+        }),
+        authManager: { getUser: jest.fn().mockResolvedValue(mockUserImx) } as unknown as AuthManager,
+      });
```

Comment:
Can we use `jest.clearAllMocks` here, rather than having to reinitialise the `guardianClient` again?